### PR TITLE
ci: validate fix

### PR DESCRIPTION
**Pull request by blaggacao**
Alternative to https://github.com/frappe/erpnext/pull/43517

cc @rohitwaghchaure This should be the definitive temporary work around, as it seems the other one was not quite enough. :pray:

I'm confident, because this was the only functional change introduced to `parallel_test_runner` code path

### CODE REVIEW

File: .github/workflows/server-tests-mariadb.yml [hunk 1]
```diff
@@ -116,8 +116,8 @@ jobs:
         env:
           DB: mariadb
           TYPE: server
-          FRAPPE_USER: ${{ github.event.inputs.user }}
-          FRAPPE_BRANCH: ${{ github.event.inputs.branch }}
+          FRAPPE_USER: blaggacao
```
Comment (ruthra-kumar) [inline .github/workflows/server-tests-mariadb.yml:119]: Could you explain why this change?

**Review by ruthra-kumar — APPROVED** [review]

File: .github/workflows/server-tests-mariadb.yml (hunk 1 above)
Comment (blaggacao) [reply .github/workflows/server-tests-mariadb.yml:119]: This is just for downstream validation of an upstream PR.

(And because it was successful, I merged it and we can close this PR)

**Comment by blaggacao** [comment]
Validation succeed: https://github.com/frappe/frappe/pull/28006 merged.
